woocommerce/woocommerce-gutenberg-products-block
# [Store Customization] Re-enable the `Product Hero 2 Column 2 Row` pattern

## What

This PR adds back the `Product Hero 2 Column 2 Row` with images.

## Why

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11236


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert the `Product Hero 2 Column 2 Row` pattern.
3. Check it has images and it renders correctly in the editor and the front end.


## Screenshots or screencast


![CleanShot 2023-10-19 at 10 26 13@2x](https://github.com/woocommerce/woocommerce-blocks/assets/186112/186796fb-c05e-403b-91d0-aeb8e13fd466)


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add back the `Product Hero 2 Column 2 Row` pattern.
